apps/condo/domains/banking/utils/serverSchema/converters/convertFrom1CExchangeToSchema.js

```diff
@@ -0,0 +1,272 @@
+const readline = require('readline')
+
+const dayjs = require('dayjs')
+const customParseFormat = require('dayjs/plugin/customParseFormat')
+dayjs.extend(customParseFormat)
+
+const DATE_FORMAT = 'DD.MM.YYYY'
+
+/**
+ * Parses `key=value` string to JSON
+ */
+function getKeyValueFrom (line) {
+    return line.split('=')
+}
+
+/**
+ * Represents entity, that starts at "begin" marker-line, has key-value body and ends with "end" marker
+ */
+class StreamNode {
+    // TODO(antonal): validate with Ajv after collecting all keys. Replace `bodyKeys` with Ajv validation schema
+    /**
+     *
+     * @param {String/Regexp} begin - line, that determines beginning of this entity (exact match)
+     * @param {String} [end] - line, that determines end-border of this entity (exact match)
+     * @param {String} [endBodyKey] – some entities does not have end markers, but have last keys. After parsing this key, entity will be finished
+     * @param {String[]} bodyKeys
+     */
+    constructor ({ begin, end, endBodyKey, bodyKeys, converter }) {
+        this.begin = begin
+        this.end = end
+        this.endBodyKey = endBodyKey
+        this.bodyKeys = bodyKeys
+        this.body = {}
+        this.finished = false
+        this.valid = true
+        this.converter = converter
+    }
+
+    parse (line) {
+        if (line === this.end) {
+            this.finished = true
+        } else {
+            const [key, value] = getKeyValueFrom(line)
+            if (this.bodyKeys.includes(key)) {
+                this.body[key] = value
+                if (this.endBodyKey === key) {
+                    this.finished = true
+                }
+            } else {
+                this.valid = false
+            }
+        }
+    }
+
+    convert () {
+        return this.converter(this.body)
+    }
+
+    get isFinished () {
+        return this.finished
```

Comment:
If you're define getter, I would prefer to make this field private with `#` notation
```suggestion
        return this.#finished
```